testing-guidance/workflow-tests/track-view/track-view-tests.md

```diff
@@ -0,0 +1,88 @@
+## **Track View Workflow Tests**
+
+Testing in this area should focus on the functionality of the Track View. This is a general tool for the creating, editing and exporting of cutscenes.
```

Comment:
nit: 'the Track View' -> 'Track View'

nit: 'for the creating' -> 'for the creation'